src/events/guildMemberUpdate.js

```diff
@@ -0,0 +1,56 @@
+const { getAuditChannel } = require("../utils/functions");
+const { MessageEmbed } = require('discord.js');
+const Discord = require('discord.js')
+module.exports = {
+    name: "guildMemberUpdate",
+    async execute(bot, newMember, oldMember) {
+            const auditChannel = await getAuditChannel(newMember.guild.id);
+        
+            // not enabled
+            if (auditChannel === null || !auditChannel) return;
+        const embed = new MessageEmbed()
+    .setAuthor(`${newMember.user.tag}`, newMember.user.displayAvatarURL({ dynamic: true }))
+    .setTimestamp()
+    .setColor(oldMember.guild.me.displayHexColor);
```

Comment:
Can you update the color to `ORANGE` to keep consistency across the other update events

Reply:
> Please view requested. Do you mind to format the code too :) please, thanks!

what?
i dont get your question?